### PR TITLE
[v7r0] Add pilot option -z in _getPilotOptions

### PR DESCRIPTION
**Pull request by wkrzemien**
BEGINRELEASENOTES

Please follow the template:
*WorkloadManagementSystem
NEW
By default the option is not present (desactivated).
To activate the pilot logging mechanism the following option
must be present in CS: Pilot/PilotLogging with a value set to
"true", "yes" or "y".

ENDRELEASENOTES

### CODE REVIEW

File: WorkloadManagementSystem/Agent/SiteDirector.py [hunk 1]
```diff
@@ -1099,6 +1099,11 @@ def _getPilotOptions(self, queue, pilotsToSubmit, **kwargs):
     else:
       self.log.info('DIRAC project will be installed by pilots')
 
+    # Pilot Logging defined?
+    pilotLogging = opsHelper.getValue("Pilot/PilotLogging", "false")
+    if pilotLogging.lower() in ['true', 'yes', '1']:
```
Comment (fstagni) [inline WorkloadManagementSystem/Agent/SiteDirector.py]:
```suggestion
    if pilotLogging.lower() in ['true', 'yes', 'y']:
```

**Comment by fstagni** [comment]
You can target `rel-v7r0`

**Comment by fstagni** [comment]
Up! Can you target rel-v7r0 instead of integration please? And then also update https://github.com/DIRACGrid/DIRAC/wiki/DIRAC-v7r0

**Comment by wkrzemien** [comment]
Closing cause this one #4343  is targeting the correct branch

**Pull request closed without merging**